tests/test_course.py

```diff
@@ -21,6 +21,7 @@
 from canvasapi.exceptions import RequiredFieldMissing, ResourceDoesNotExist
 from canvasapi.external_feed import ExternalFeed
 from canvasapi.external_tool import ExternalTool
+from canvasapi.lti_resource_link import LTIResourceLink
```

Comment:
There's a check on this project that imports need to be alphabetically sorted, so this needs to be moved down to be under
`from canvasapi.license import License`

Reply:
Thanks for noting, just fixed in most recent commit!